presto-main/src/main/java/io/prestosql/operator/window/ReflectionWindowFunctionSupplier.java

```diff
@@ -29,7 +30,8 @@
 public class ReflectionWindowFunctionSupplier<T extends WindowFunction>
         extends AbstractWindowFunctionSupplier
 {
-    private final Constructor<T> constructor;
+    private Constructor<T> constructor;
```

Comment:
Keep these as `final` so the class is immutable. The constructor can be structured so that this is possible (probably need a temporary local variable)